partiql-ast/src/main/java/org/partiql/ast/expr/TruthValue.java

```diff
@@ -0,0 +1,62 @@
+package org.partiql.ast.expr;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.ast.AstEnum;
+import org.partiql.ast.AstNode;
+import org.partiql.ast.AstVisitor;
+
+import java.util.List;
+
+/**
+ * TODO docs
+ */
+public class TruthValue extends AstEnum {
+    public static final int UNKNOWN = 0;
+    public static final int TRUE = 1;
+    public static final int FALSE = 2;
+    public static final int UNK = 3;
```

Comment:
Interesting! Maybe OTHER, or maybe we don't need to define the `UNK` variant for this. It would still force non-exhaustive and we have -1 reserved. Maybe you make it

```
// implicit uknown is 0
TRUE = 1
FALSE = 2
UNKNOWN = 3
```

Reply:
I wasn't too sure what to do w/ the name collision. Ideally for the sake of consistency, I would like to keep the unknown/other variant the same across `AstEnum`'s.

Perhaps we could rename all of the unknown/other variants something that won't have a collision like `_UNKNOWN` or `_OTHER`? The static functions could then be named `_UNKNOWN()`/`_OTHER()`.

Reply:
I don't think I have a preference on the exact name so let's check in with John on Tuesday's meeting. I think the important thing is avoiding current and future collisions with what you have suggested.

Reply:
Yeah, I've been thinking about this. If there is good separation between our library APIs and serialization, then I _believe_ we can remove the UNKNOWN variants of the other enums.

Reply:
Yeah, let's discuss this tomorrow. I'll need to check w/ the original rationale for including an `UNKNOWN` variant for all of the enums. Perhaps it's required for serde? But we could always add the variant and relevant functions back in the future.